src/main/resources/config/application-prod.yml

```
@@ -119,8 +125,10 @@ info:
             - programming_exercise_fail_tour: 'tutorial'
             - programming_exercise_success_tour: 'tutorial'
             - tutor_assessment_tour: 'Patterns in Software Engineering'
-    contact: artemis@xcit.tum.de                                                                  #default value, can be overridden on the server
     test-server: false  # false --> production, true --> test server, --> empty == local
     # Specifies whether text assessment analytics service (TextAssessmentEventResource) is enabled/disabled
     # default value set to false in production
     text-assessment-analytics-enabled: false
+    operatorName: # Must be set before starting the application in production. Shown in the about us page and sent to the telemetry service (e.g. university or school)
+    operatorAdminName: # Can be set before starting the application in production. Shown in the about us page and used by the telemetry service
+    contact: # The admins contact email address, shown in the about us page, and sent to the telemetry service
```

```suggestion
    # operatorName: # Must be set before starting the application in production. Shown in the about us page and sent to the telemetry service (e.g. university or school)
    # operatorAdminName: # Can be set before starting the application in production. Shown in the about us page and used by the telemetry service
    # contact: # The admins contact email address, shown in the about us page, and sent to the telemetry service
```
It might default to an empty string otherwise (not sure, should be tested).

---

I tested it: `operatorName` needs to be defined anyway (with content), but for the others its fine if they are empty (or commented out). So I would leave it like it is